conf/config-default.yaml

```diff
@@ -238,6 +238,7 @@ plugins:                          # plugin list (sorted in alphabetical order)
   - uri-blocker
   - wolf-rbac
   - zipkin
+  - server-info
```

Comment:
we should disable this plugin by default

Reply:
What's the reason for disabling this plugin by default? Users are no feeling about the behavior of server-info.